clusters/build-clusters/01_cluster/openshift-image-registry/README.md

```diff
@@ -17,3 +17,12 @@ Pull image:
 # podman pull default-route-openshift-image-registry.apps.build01.ci.devcluster.openshift.com/ci/ci-operator:latest --tls-verify=false
 
 ```
+
+## Scale up
+
+[edit the image registry operator config](https://coreos.slack.com/archives/C011JMD9UB1/p1587059047037500)
```

Comment:
I think we should link to OCP documentation instead of Slack convos.

Reply:
I added both now.

I was recording what happened in the slack channel for debugging the issue.
In case we want to deploy a new cluster in build farm or redeploy build01 we have those.

However, the doc does not talk about how to scale up as we do it for others via a yaml file.
This is not automated by our applyconfig.

I was using slack links to note down why we are doing this, instead of yamls.

Reply:
NVM ... removed slack URLs.